src/main/java/com/modus/camel/datasonnet/language/DatasonnetExpression.java

```diff
@@ -1,15 +1,18 @@
 package com.modus.camel.datasonnet.language;
 
 import com.modus.camel.datasonnet.DatasonnetProcessor;
-import org.apache.camel.*;
+import org.apache.camel.CamelContext;
+import org.apache.camel.Exchange;
+import org.apache.camel.Expression;
+import org.apache.camel.RuntimeExpressionException;
 import org.apache.camel.spi.GeneratedPropertyConfigurer;
 import org.apache.camel.support.ExpressionAdapter;
 import org.apache.camel.support.component.PropertyConfigurerSupport;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class DatasonnetExpression extends ExpressionAdapter implements GeneratedPropertyConfigurer {
-    private final String expression;
+    private String expression;
```

Comment:
Why remove final here?

Reply:
Ah, I know, because it adds another constructor which doesn't initialize expression.